src/RSocketRequester.cpp

```diff
@@ -26,14 +26,6 @@ RSocketRequester::~RSocketRequester() {
   }
 }
 
-void RSocketRequester::closeSocket() {
```

Comment:
That is a good point. I did not know about the usage. 
What if we don't create Requester on the server - should we have ways to close individual instance from RSocketResponder as well?  In general I think Requester/Responder should just request/respond.  Should we add ability in RSocketServer to close specific instances ?